workers/unity/Packages/com.improbable.gdk.mobile/Utility/DeviceInfo.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile
+{
+    public static class DeviceInfo
+    {
+        public const string AndroidEmulatorDefaultCallbackIp = "10.0.2.2";
```

Comment:
What is this ip?

Reply:
the ip necessary to connect your emulator to your localhost